text/0034-telemetry-policies.md

```diff
@@ -0,0 +1,111 @@
+- Title: Telemetry Policies
+- Start Date: 2021-04-16
+- Specification PR: [#34](https://github.com/meilisearch/specifications/pull/34)
+- MeiliSearch Tracking-Issues: TBD
+
+# Telemetry Policies
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Telemetry concerns the possibility of depositing markers within a system in order to make segments of it. Quantitative data as data metrics are used to discover and verify user behavior.
+
+We would like to have more telemetry to better understand our users in order to improve their usage and feel with Meilisearch.
+
+### II. Motivation
+
+We need to make informed decisions based on data instead of instincts and gut feel. Lack of data does not say anything about how correct or wrong are our intuitions. Lack of data means we did not make testable hypothesis. We will now support ours decisions with data.
+
+> 🔒 Note that we strictly don't want to collect private information by default (ip address, email, website url, etc..).
+
+### III. Additional Materials
+N/A
+
+### IV.Explanation
+
+Following this issue and with the beginning of a product Team we asked yourselves what we wanted to do about telemetry. https://github.com/meilisearch/transplant/issues/141.
+
+✅ We will keep telemetry enabled by default while leaving the option for users to disable it. See Impact on Documentation part.
+
+> Some new features will had telemetry data points to check the assumptions and measure objectives achievments.
+
+✅ Each new feature specification should list data points to be collected and measured. Obviously the specification must explain why these data points are necessary in relation to the feature.
+
+#### Current stack for analytics
+
+We are currently using Amplitude as a quantitative analytic tool to measure usage as a big picture.
+
+E.g.
+- Cohort: Servers that are running for more than seven days with the old version of MS (< v0.10.1)
+- Cohort: Servers that are running for more than seven days with the new version of MS (> v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the old version of MS (< v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the new version of MS (> v0.10.1)
+- Graph: The total of every telemetry messages received
+- Graph: The total of every unique id seen
+- Graph: The number of servers that are running for more than 7 days/weekly
+- Graph: The number of servers that are running for more than 30 days/month
```

Comment:
Dummy question (not an expert in analytics at all): what happen for February? I mean, are the analytics for this month broken?

Reply:
Hello @curquiza! This is not a problem, this is an accumulation of servers running in production distributed by months !